upgrade to sonobuoy 0.19.0

paynejacob opened the pull request:
https://github.com/rancher/rancher/issues/33335

Upgraded sonobuoy to v 0.19.0

paynejacob commented:
There will be a companion pr for the chart to add this update, I will wait to create it as it is just version bumps.

prachidamle commented:
@paynejacob we need to test all cis scans against the supported  k8s versions with this Sonobuoy version to see if it supports everything

paynejacob commented:
@prachidamle I will test the versions and post the results here.  is the permissive scan sufficient or should I create hardened clusters as well?

paynejacob commented:
# Sonobuoy 0.19.0
```
Switched to context "jpayne-cis-17".
NAME                CLUSTERSCANPROFILE           TOTAL   PASS   FAIL   SKIP   WARN   NOT APPLICABLE   LASTRUNTIMESTAMP       CRONSCHEDULE
rke-hardened-15     rke-profile-hardened-1.5     122     63     13     0      25     21               2021-07-21T22:02:45Z
rke-hardened-16     rke-profile-hardened-1.6     122     65     12     0      24     21               2021-07-21T22:02:24Z
rke-permissive-15   rke-profile-permissive-1.5   122     62     0      0      24     36               2021-07-21T22:02:14Z
rke-permissive-16   rke-profile-permissive-1.6   122     62     0      0      24     36               2021-07-21T22:02:03Z
rke-profile-15      cis-1.5-profile              122     51     32     0      39     0                2021-07-21T22:03:26Z
rke-profile-16      cis-1.6-profile              122     56     29     0      37     0                2021-07-21T22:04:48Z
Switched to context "jpayne-cis-18".
NAME                CLUSTERSCANPROFILE           TOTAL   PASS   FAIL   SKIP   WARN   NOT APPLICABLE   LASTRUNTIMESTAMP       CRONSCHEDULE
rke-hardened-15     rke-profile-hardened-1.5     122     63     13     0      25     21               2021-07-21T22:02:47Z
rke-hardened-16     rke-profile-hardened-1.6     122     65     12     0      24     21               2021-07-21T22:02:27Z
rke-permissive-15   rke-profile-permissive-1.5   122     62     0      0      24     36               2021-07-21T22:02:16Z
rke-permissive-16   rke-profile-permissive-1.6   122     62     0      0      24     36               2021-07-21T22:02:06Z
rke-profile-15      cis-1.5-profile              122     51     32     0      39     0                2021-07-21T22:03:28Z
rke-profile-16      cis-1.6-profile              122     56     29     0      37     0                2021-07-21T22:04:51Z
Switched to context "jpayne-cis-19".
NAME                CLUSTERSCANPROFILE           TOTAL   PASS   FAIL   SKIP   WARN   NOT APPLICABLE   LASTRUNTIMESTAMP       CRONSCHEDULE
rke-hardened-15     rke-profile-hardened-1.5     122     63     13     0      25     21               2021-07-21T22:03:30Z
rke-hardened-16     rke-profile-hardened-1.6     122     65     12     0      24     21               2021-07-21T22:02:49Z
rke-permissive-15   rke-profile-permissive-1.5   122     62     0      0      24     36               2021-07-21T22:02:18Z
rke-permissive-16   rke-profile-permissive-1.6   122     62     0      0      24     36               2021-07-21T22:02:08Z
rke-profile-15      cis-1.5-profile              122     51     32     0      39     0                2021-07-21T22:02:30Z
rke-profile-16      cis-1.6-profile              122     56     29     0      37     0                2021-07-21T22:04:54Z
Switched to context "jpayne-cis-20".
NAME                CLUSTERSCANPROFILE           TOTAL   PASS   FAIL   SKIP   WARN   NOT APPLICABLE   LASTRUNTIMESTAMP       CRONSCHEDULE
rke-hardened-15     rke-profile-hardened-1.5     122     63     13     0      25     21               2021-07-21T22:02:51Z
rke-hardened-16     rke-profile-hardened-1.6     122     65     12     0      24     21               2021-07-21T22:02:31Z
rke-permissive-15   rke-profile-permissive-1.5   122     62     0      0      24     36               2021-07-21T22:02:20Z
rke-permissive-16   rke-profile-permissive-1.6   122     62     0      0      24     36               2021-07-21T22:02:10Z
rke-profile-15      cis-1.5-profile              122     51     32     0      39     0                2021-07-21T22:03:33Z
rke-profile-16      cis-1.6-profile              122     56     29     0      37     0                2021-07-21T22:04:55Z
```

# Sonobuoy 0.16.3
```
Switched to context "jpayne-cis-17".
NAME                CLUSTERSCANPROFILE           TOTAL   PASS   FAIL   SKIP   WARN   NOT APPLICABLE   LASTRUNTIMESTAMP       CRONSCHEDULE
rke-hardened-15     rke-profile-hardened-1.5     122     63     13     0      25     21               2021-07-21T22:11:59Z
rke-hardened-16     rke-profile-hardened-1.6     122     65     12     0      24     21               2021-07-21T22:09:35Z
rke-permissive-15   rke-profile-permissive-1.5   122     62     0      0      24     36               2021-07-21T22:09:54Z
rke-permissive-16   rke-profile-permissive-1.6   122     62     0      0      24     36               2021-07-21T22:09:13Z
rke-profile-15      cis-1.5-profile              122     51     32     0      39     0                2021-07-21T22:10:37Z
rke-profile-16      cis-1.6-profile              122     56     29     0      37     0                2021-07-21T22:14:42Z
Switched to context "jpayne-cis-18".
NAME                CLUSTERSCANPROFILE           TOTAL   PASS   FAIL   SKIP   WARN   NOT APPLICABLE   LASTRUNTIMESTAMP       CRONSCHEDULE
rke-hardened-15     rke-profile-hardened-1.5     122     63     13     0      25     21               2021-07-21T22:12:00Z
rke-hardened-16     rke-profile-hardened-1.6     122     65     12     0      24     21               2021-07-21T22:10:37Z
rke-permissive-15   rke-profile-permissive-1.5   122     62     0      0      24     36               2021-07-21T22:09:56Z
rke-permissive-16   rke-profile-permissive-1.6   122     62     0      0      24     36               2021-07-21T22:09:15Z
rke-profile-15      cis-1.5-profile              122     51     32     0      39     0                2021-07-21T22:14:44Z
rke-profile-16
Switched to context "jpayne-cis-19".
NAME                CLUSTERSCANPROFILE           TOTAL   PASS   FAIL   SKIP   WARN   NOT APPLICABLE   LASTRUNTIMESTAMP       CRONSCHEDULE
rke-hardened-15     rke-profile-hardened-1.5     122     63     13     0      25     21               2021-07-21T22:12:02Z
rke-hardened-16     rke-profile-hardened-1.6     122     65     12     0      24     21               2021-07-21T22:10:40Z
rke-permissive-15   rke-profile-permissive-1.5   122     62     0      0      24     36               2021-07-21T22:09:58Z
rke-permissive-16   rke-profile-permissive-1.6   122     62     0      0      24     36               2021-07-21T22:09:17Z
rke-profile-15      cis-1.5-profile              122     51     32     0      39     0                2021-07-21T22:14:46Z
rke-profile-16
Switched to context "jpayne-cis-20".
NAME                CLUSTERSCANPROFILE           TOTAL   PASS   FAIL   SKIP   WARN   NOT APPLICABLE   LASTRUNTIMESTAMP       CRONSCHEDULE
rke-hardened-15     rke-profile-hardened-1.5     122     63     13     0      25     21               2021-07-21T22:12:04Z
rke-hardened-16     rke-profile-hardened-1.6     122     65     12     0      24     21               2021-07-21T22:09:41Z
rke-permissive-15   rke-profile-permissive-1.5   122     62     0      0      24     36               2021-07-21T22:10:01Z
rke-permissive-16   rke-profile-permissive-1.6   122     62     0      0      24     36               2021-07-21T22:09:19Z
rke-profile-15      cis-1.5-profile              122     51     32     0      39     0                2021-07-21T22:10:43Z
rke-profile-16      cis-1.6-profile              122     56     29     0      37     0                2021-07-21T22:14:48Z
```

test results are the same as before these changes

prachidamle commented on package/run.sh:
```diff
@@ -54,7 +54,7 @@ echo "Rancher: Running CIS Benchmarks"
 
 # Run sonobuoy first
-if ! sonobuoy master -v 3
+if ! sonobuoy aggregator -v 3
```
Did the sonobuoy command change?

paynejacob replied:
yes there was a pr to remove insensitive terms https://github.com/vmware-tanzu/sonobuoy/commit/046c0827bd950c5c0117812fb76c9854edea5279

prachidamle replied:
ok, thanks

prachidamle reviewed (approved):
LGTM, need a charts PR update
